server.py

```diff
@@ -360,6 +360,41 @@ def healthcheck():
         return jsonify({'status': 'OK'})
 
 
+def get_all_comments_by_survey_id(survey_id):
+    # collect survey based on survey id and and 146 code
+
+    records = db.session.query(SurveyResponse).filter(SurveyResponse.data['survey_id'].astext == survey_id).all()
+    logger.info("Comments retrieved " + str(len(records)))
+    return records
+
+@app.route('/hallo', methods=['GET'])
+def get_hallo():
+    return jsonify({'status': 'OK'})
+
+
+@app.route('/comments/<string:survey_id>', methods=['GET'])
+def get_comments(survey_id):
+    logger.info("Exporting comments for survey id ", survey_id=survey_id)
+
+    if not survey_id:
+        logger.error("Survey_id is none : " + survey_id)
+        return server_error(400)
+
+    logger.info("Exporting comments for survey id " + survey_id)
+    try:
+        comments = get_all_comments_by_survey_id(survey_id)
+
+        if not comments or len(comments) == 0:
+            return jsonify({'No comments to export for ': survey_id})
+
+        workbook = exporter.create_comments_book(survey_id, get_all_comments_by_survey_id(survey_id))
+    except Exception as e:
```

Comment:
Best practice is to catch more specific exceptions where you can in this case you'd probably want `SQLAlchemyError` and something for if the workbook stuff errors. Not sure what you catch for that but I found this funny(/worrying) comment on stackoverflow when I was looking into it 
<img width="679" alt="screen shot 2018-07-05 at 15 05 14" src="https://user-images.githubusercontent.com/22077099/42327988-16496368-8065-11e8-9b03-45e42909b558.png">

Reply:
not sure for such temp functionality we neeed to get that deep in a specific error management, that to be honest does not give me much value. You might get DB errors, or IO error, but because get run just from us once in a while possibly from local machine i felt like an overkill.

Reply:
Cool makes sense to me, If this is just a temporary solution that should be fine